functions/pe_db_names.pp

```diff
@@ -9,10 +9,18 @@ function peadm::pe_db_names (
     'pe-rbac',
   ]
 
+  $pe_2025_3_or_later = SemVerRange('>= 2025.3.0')
   $pe_2025_or_later = SemVerRange('>= 2025.0.0')
   $pe_2023_8_or_later = SemVerRange('>= 2023.8.0')
 
   case $pe_ver {
+    # The infra-assistant was added in 2025.3.0
+    $pe_2025_3_or_later: {
+      $original_db_names + [
+        'pe-infra-assistant',
```

Comment:
Puppet case statements don't fall through so I think you need to include all of the previous dbs: pe-hac and pe-patching here as well.

Reply:
Ah crap, thanks for catching that.